apps/sync/js/fxsync-webcrypto/fxsyncwebcrypto.js

```diff
@@ -0,0 +1,279 @@
+'use strict';
+
+/* global StringConversion, KeyDerivation, crypto */
+/* exported FxSyncWebCrypto */
+
+// WebCrypto-based client for Firefox Sync.
```

Comment:
You have an excellent documentation at https://github.com/michielbdejong/fxsync-webcrypto/blob/initial-implementation/doc/how-it-works.md It might be worth adding part of it here in the code. Or maybe add the .md files directly to the apps/sync/ directory.